runner.go

```diff
@@ -171,32 +258,36 @@ func StartServer() {
 
 	http.Handle("/livez", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		if cmd.ProcessState != nil && cmd.ProcessState.Exited() {
-			w.WriteHeader(http.StatusInternalServerError)
-			w.Write([]byte("not ok"))
+			writeResponse(w, http.StatusInternalServerError, "not ok")
 		} else {
-			w.WriteHeader(http.StatusOK)
-			w.Write([]byte("ok"))
+			writeResponse(w, http.StatusOK, "ok")
 		}
 	}))
 
 	http.Handle("/healthz", http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		if health.Check() {
-			w.WriteHeader(http.StatusOK)
-			w.Write([]byte("ok"))
+			writeResponse(w, http.StatusOK, "ok")
 		} else {
-			w.WriteHeader(http.StatusInternalServerError)
-			w.Write([]byte("not ok"))
+			writeResponse(w, http.StatusInternalServerError, "not ok")
 		}
 		nok := map[bool]string{false: "not ok", true: "ok"}
 		if r.URL.Query().Has("verbose") {
 			for k, v := range health.Flatten() {
-				w.Write([]byte("\n[+]" + k + " " + nok[v.Healthy()]))
+				if _, err := w.Write([]byte("[+]" + k + " " + nok[v.Healthy()] + "\n")); err != nil {
+					log.Print("failed writing response: ", err.Error())
+					break
+				}
 			}
-			w.Write([]byte("\n"))
 		}
 	}))
 
-	http.ListenAndServe("0.0.0.0:8090", http.DefaultServeMux)
+	l, err := net.Listen("tcp", ":8090")
```

Comment:
This is throwing a lint error for `G102: Binds to all network interfaces`. More info [here](https://securego.io/docs/rules/g102.html) but I'm not sure how to address (heh) the problem. We might be able to pass the listen address through an environment variable.